lib/biran/config_defaults.rb

```diff
@@ -54,6 +54,9 @@ def secrets_file
     end
 
     def default_db_config_file
+      unless defined? Rails
+        return "#{app_defaults_init[:app][:base_path]}/#{configuration.config_dirname}/#{configuration.db_config_filename}"
+      end
```

Comment:
Isn't this already handled in the configuration object?
```
     def base_path
       return @base_path if @base_path
       @base_path = Rails.root if defined? Rails
       @base_path ||= './'
     end
```
Do we need the unless block? Can we just  use the return string  and drop the Rails.root line below?

Reply:
No

Reply:
Is that no we don't need the block or is it no we can't use the return string.
Also, I think this would be better anyway
```
    def default_db_config_file
      File.join(config_dir, configuration.db_config_filename)
    end
```
But that is untested.

Reply:
Let me check this one, probably not needed.

Reply:
I believe the File.join line I have above works as expected. I have done some basic test against another app that uses  rails and the database.yml file with my proposed set of minimal changes and it seems to work as expected.

Reply:
Is not working as expected. Once I remove this, running the task raises the exception `NameError: uninitialized constant Biran::ConfigDefaults::Rails`